Project/Assets/ML-Agents/Examples/Walker/Scripts/WalkerAgent.cs

```diff
@@ -101,10 +115,11 @@ public void CollectObservationBodyPart(BodyPart bp, VectorSensor sensor)
     /// </summary>
     public override void CollectObservations(VectorSensor sensor)
     {
-        sensor.AddObservation(Quaternion.FromToRotation(hips.forward, orientationCube.transform.forward));
-        sensor.AddObservation(Quaternion.FromToRotation(head.forward, orientationCube.transform.forward));
+        sensor.AddObservation(hips.rotation);
+        sensor.AddObservation(head.rotation);
```

Comment:
I thought you tested and `Quaternion.FromToRotation(hips.forward, orientationCube.transform.forward)` worked better than `hips.rotation` ?

Reply:
this was an oversight from some testing i was doing. i changed it back to the previous observations.